Makefile

```diff
@@ -185,107 +184,227 @@ test-integration: test-integration-aws test-integration-clickhouse test-integrat
 test-integration: test-integration-gcp test-integration-influxdb test-integration-kafka test-integration-loki
 test-integration: test-integration-pulsar test-integration-splunk
 
+stop-test-integration: ## Stops all integration test infrastructure
+stop-test-integration: stop-integration-aws stop-integration-clickhouse stop-integration-elasticsearch
+stop-test-integration: stop-integration-gcp stop-integration-influxdb stop-integration-kafka stop-integration-loki
+stop-test-integration: stop-integration-pulsar stop-integration-splunk
+
+start-integration-aws:
+	$(CONTAINER_TOOL) network create test-integration-aws
+	$(CONTAINER_TOOL) run -d --network=test-integration-aws -p 8111:8111 --name ec2_metadata timberiodev/mock-ec2-metadata:latest
+	$(CONTAINER_TOOL) run -d --network=test-integration-aws -p 4568:4568 -p 4572:4572 -p 4582:4582 -p 4571:4571 -p 4573:4573 --name localstack -e SERVICES=kinesis:4568,s3:4572,cloudwatch:4582,elasticsearch:4571,firehose:4573 localstack/localstack@sha256:f21f1fc770ee4bfd5012afdc902154c56b7fb18c14cf672de151b65569c8251e
+	$(CONTAINER_TOOL) run -d --network=test-integration-aws -p 6000:6000 --name mockwatchlogs -e RUST_LOG=trace luciofranco/mockwatchlogs:latest
+	sleep 60 # Many services are very slow... Give them a sec...
```

Comment:
I feel like 60 is a lot?

Reply:
I don't have a lot of context here, but could we do something like the `wait_for_tcp` func we have in tests to check that ports are open?

I know that 'open port' != 'available' in some cases so maybe this doesn't accommodate all cases, but a static 1 min sleep might be 50 seconds of waiting around.

Reply:
Can pretty much guarantee it's not. :) The major thing stopping us seeing a lot of failed tests due to infrastructure has been the cargo build process, it that takes a long enough time that services are up and running by the time the tests run.

I've looked at healthchecking - most of the images don't have basic tools on them like nc et al. The ones that do - often the port being open isn't actually representative of the service being ready.

I am going to poke some more.